src/pack.rs

```diff
@@ -350,19 +351,21 @@ impl Packer {
                 .info(&format!("total read       : {:.2}", Size(total_read)));
             self.output.report.info(&format!(
                 "fills size       : {:.2}",
-                Size(handler.fill_size)
+                Size(handler.stats.fill_size)
             ));
             self.output.report.info(&format!(
                 "duplicate data   : {:.2}",
-                Size(total_read - handler.data_written - handler.fill_size)
+                Size(total_read - handler.stats.data_written - handler.stats.fill_size)
```

Comment:
This equation should be adjusted. Now it's easier to get this value underflow.

Reply:
Done